symfony/plugins/orangehrmTimePlugin/Dao/TimesheetDao.php

```diff
@@ -226,28 +226,18 @@ public function getTimesheetActionLogById($timesheetActionLogId)
     }
 
     /**
-     * Get Timesheet by given Start Date And Employee Id
-     * @param $starDate , $employeeId
-     * @return Timesheet
+     * @param DateTime $startDate
+     * @param int $employeeId
+     * @return Timesheet[]
      */
-    public function getTimesheetByStartDateAndEmployeeId($startDate, $employeeId)
+    public function getTimesheetByStartDateAndEmployeeId(DateTime $startDate, int $employeeId): array
     {
-        // TODO
-        try {
-            $query = Doctrine_Query::create()
-                    ->from("Timesheet")
-                    ->where("start_date = ?", $startDate)
-                    ->andWhere("employee_id = ?", $employeeId);
-
-            $results = $query->execute();
-            if ($results[0]->getTimesheetId() == null) {
-                return null;
-            } else {
-                return $results[0];
-            }
-        } catch (Exception $ex) {
-            throw new DaoException($ex->getMessage());
-        }
+        $q = $this->createQueryBuilder(Timesheet::class, 'timesheet');
+        $q->andWhere('timesheet.startDate = :startDate');
+        $q->andWhere('timesheet.employee = :employeeId');
+        $q->setParameter('startDate', $startDate);
+        $q->setParameter('employeeId', $employeeId);
```

Comment:
```suggestion
        $q->setParameter('empNumber', $employeeId);
```

Reply:
Method removed because we have used from detailedTimesheet object in service